src/main/resources/Macros/MapMacro/MapMacro.xml

```diff
@@ -20,15 +20,15 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.3" reference="XWiki.MapMacro" locale="">
-  <web>XWiki</web>
+<xwikidoc version="1.3" reference="Macros.MapMacro.MapMacro" locale="">
+  <web>Macros.MapMacro</web>
```

Comment:
`Macros.MapMacro.MapMacro` doesn't look too good. What do you say about renaming it to `Macros.MapMacro.WebHome` and delete the WebHome you created? It does not make any sense to have an empty WebHome.

Reply:
I renamed it accordingly indeed.